src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+@use '../_variables/variables' as *;
+
+.page {
+  font-family: $PrimaryFont;
+  color: $ColorMainAccent;
+  font-weight: $FontRegular;
```

Comment:
The variable `$FontRegular` should follow the naming convention for SCSS variables, which typically use lowercase letters and hyphens. Consider renaming it to `$font-regular`.